test/bastion_linux_test.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestS3Creation(t *testing.T) {
+	t.Parallel()
+
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "./unit-test",
+	})
+
+    // Would fail as lifecycle.prevent_destroy is set on the bucket
+	//defer terraform.Destroy(t, terraformOptions)
```

Comment:
I've just messaged Steve about changing the bucket so that it can be destroyed - https://mojdt.slack.com/archives/C013RM6MFFW/p1655221495121939?thread_ts=1655219520.314689&cid=C013RM6MFFW

We'll need some way to destroy it otherwise it's just going to sit in the testing account.  Possibly could use Nuke or the SDK to delete the bucket then remove from TF state.  Let's discuss at stand up.

Reply:
I think we are going to remove the prevent destroy option on the bucket.